a10_octavia/tests/unit/controller/worker/tasks/test_a10_database_tasks.py

```diff
@@ -526,3 +526,28 @@ def test_CheckExistingVthunderTopology_execute_raised_exception(self):
         lb_task.vthunder_repo = mock.MagicMock()[1:999]
         lb_task.vthunder_repo.get_vthunder_by_project_id.return_value = vthunder
         self.assertRaises(o_exceptions.InvalidTopology, lb_task.execute, LB, "SINGLE")
+
+    def test_GetListenerListInLoadbalancer(self):
```

Comment:
https://github.com/a10networks/a10-octavia/pull/471/files#diff-3b2e28554ff3c3e18debbd4916f882ef14bea43f73b73b9e6361bdb6c1b0e9acR77

Not sure what happened to the database unit tests. They seem to be out of sync with our unit test names. But this is how they should be named. Please add more info on these two tests

Reply:
Done